flashlib.core.robot/src/main/java/com/flash3388/flashlib/robot/control/PidController.java

```diff
@@ -173,32 +140,35 @@ public void setOutputLimit(double outputLimit){
      * Calculates the output to the system to compensate for the error.
      *
      * @param processVariable the process variable of the system.
-     * @param setPoint the desired set point.
+     * @param setpoint the desired set point.
      *
      * @return the compensation value from the PID loop calculation
      */
-    public double calculate(double processVariable, double setPoint){
+    public double applyAsDouble(double processVariable, double setpoint, Time currentTimeSeconds) {
```

Comment:
`currentTimeSeconds` -> `currentTime`.
Also, update the javadoc with the new parameter.

Reply:
yeah yeah